pydantic_ai_slim/pydantic_ai/agent/__init__.py

```diff
@@ -1470,6 +1473,65 @@ def _set_sampling_model(toolset: AbstractToolset[AgentDepsT]) -> None:
 
         self._get_toolset().apply(_set_sampling_model)
 
+    def to_web(
+        self,
+        *,
+        models: list[AIModel] | None = None,
+        builtin_tool_defs: list[BuiltinToolDef] | None = None,
+    ) -> FastAPI:
```

Comment:
I had a comment elsewhere about including more arguments here that can be passed through to the the Starlette app (see `to_ag_ui` for example).

Also we should use plain Starlette if we can, not FastAPI!